pkgs/tools/inputmethods/interception-tools/interception-k2k.nix

```
@@ -0,0 +1,26 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "interception-k2k";
+  version = "0-unstable-2024-11-06";
+
+  src = fetchFromGitHub {
+    owner = "zsugabubus";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "interception-k2k";
```

(Nit) `repo = pname` is an anti-pattern

Otherwise LGTM